src/lib.rs

```diff
@@ -1508,7 +1508,8 @@ impl PublicKeyData for KeyPair {
 	}
 }
 
-trait PublicKeyData {
+#[allow(missing_docs)]
+pub trait PublicKeyData {
```

Comment:
Why is making this public required?

Reply:
I don't know why the methods kept to be private. All methods in the trait are immutable.